worlds/models/circuit/model.sdf

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<sdf version="1.4">
+    <model name="circuit">
+        <static>true</static>
+
+        <link name="link">
+            <collision name="collision">
+                <geometry>
+                    <plane>
+                        <normal>0 0 1</normal>
+                        <size>1000 1000</size>    <!-- Since is ground, make collision bigger than visual -->
+                    </plane>
+                </geometry>
+                <surface>
+                    <friction>
+                        <ode>
+                            <mu>100</mu>
+                            <mu2>50</mu2>
+                        </ode>
+                    </friction>
+                </surface>
+            </collision>
+
+            <visual name="visual">
+                <cast_shadows>false</cast_shadows>
+                <geometry>
+                    <plane>
+                        <normal>0 0 1</normal>
+                        <size>5.25 3.9375</size>    <!-- "size" based on image aspect ratio -->
```

Comment:
Is this size based on your image aspect ratios for the different textures? If not (that is you are just using values I put for the sumo ring), you need to update this to reflect the aspect ratio of the image you used for this texture (or else dimensions would be warped)

Reply:
changed size to `<size>5.25 5.25</size>` to reflect 1:1 aspect ratio